docs/codeflow/codeflow-faq.md

```diff
@@ -1,5 +1,9 @@
 ---
 title: Codeflow FAQ
+head:
+  - - meta
+    - name: og:image
+      content: https://developer.stackblitz.com/img/og/codeflow-faq.png
```

Comment:
Would it still work with domain-relative URLs, i.e.

```
content: /img/og/codeflow-faq.png
```

This would let us test social media previews locally (there are a number of browser extensions that can show previews, some dedicated extensions and some SEO ones like Ahrefs).

Reply:
That's what I was using locally to test as well.

However the config used a full url and I recall some sites have troubles with relative URLs and require the full URL.
Am I mistaken here? Would be nice if I was.

Just did a `Find + Replace` back and forth, though obviously not ideal.